cdap-examples/WikipediaPipeline/src/main/java/co/cask/cdap/examples/wikipedia/WikipediaPipelineApp.java

```diff
@@ -41,6 +41,7 @@ public void configure() {
     addStream(new Stream(RAW_WIKIPEDIA_STREAM));
     addMapReduce(new StreamToDataset(LIKES_TO_DATASET_MR_NAME));
     addMapReduce(new StreamToDataset(WIKIPEDIA_TO_DATASET_MR_NAME));
+    addMapReduce(new WikipediaDataDownloader());
```

Comment:
You've changed the Java file, but you didn't change the include. It was 24-55. Should now be 24-57, because you added two lines in that range...